libtenzir/include/tenzir/tql2/tokens.hpp

```diff
@@ -22,7 +22,7 @@ TENZIR_ENUM(
   // identifiers
   identifier, dollar_ident,
   // keywords
-  this_, if_, else_, match, not_, and_, or_, underscore, let, in, meta,
+  this_, if_, else_, match, not_, and_, or_, move, underscore, let, in, meta,
```

Comment:
Note to self: If we want to merge this, then this still needs documentation, and also a lot of examples.